tests/common/fixture.rs

```diff
@@ -40,8 +45,41 @@ pub fn verify_file(path: &Path, size: u64, seed: u64) -> Result<()> {
         .write(false)
         .custom_flags(libc::O_EXCL)
         .open(path)?;
-    let mut verifier = Verifier::new(file, seed, BLOCK_SIZE).len(size);
+    let mut verifier = Verifier::new(file, seed, BLOCK_SIZE, pattern).len(size);
     verifier.verify()
 }
 
+#[test]
+fn test_file_create_verify_repeating() -> Result<()> {
```

Comment:
Tests should not be placed in the `tests/common` directory, as it will be imported by all the test targets (pack, unpack, verify, ...) and run multiple times.

Reply:
@mingnus Ok, didn't consider that, I've moved them

Reply:
How about moving them to a stand alone file, e.g., `tests/fixture.rs` ?

Reply:
Done, thanks